README.md

```diff
@@ -241,13 +241,13 @@ You can use the Bamboo.TestAdapter along with [Bamboo.Test] to make testing your
 emails straightforward.
 
 ```elixir
-# Using the mailer from the Getting Started section
```

Comment:
This example really assumes more than the mailer from the Getting Started section, so I think it's better just not to say anything about it.